src/components/controls/dropdown/Dropdown.js

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { noop } from 'lodash';
```

Comment:
We could, but I'm sure that there's going to be more usage in the future. It's using the ``babel-lodash-plugin`` now so it's only going to pull in the functions that are used and not the entire lib.